cmd/cmd.go

```diff
@@ -329,11 +330,16 @@ An access token is required to register, and should be obtained separately from
 
 	// daemon
 	daemonCmd := appCmd.Command("daemon", "Start a node daemon session")
-	daemonRepo := daemonCmd.Flag("repo", "Specify a custom repository path").Short('r').String()
+	daemonBaseRepo := daemonCmd.Flag("base-repo", "Specify a custom path to the base repo directory").Short('b').String()
+	daemonAccountSeed := daemonCmd.Flag("account-seed", "Specify an existing acconut seed").Short('a').String()
```

Comment:
can this be address? it sounds insecure to have to paste the private key into the terminal on daemon start

Reply:
(typo in "acconut")

Reply:
Possibly. Let me look. My initial thinking was you have to be able to derive an actual `Account` (which I think you can only do from seed), but maybe an `Account` isn't required when starting the daemon.

Reply:
Either way, remember you have to provide you account seed into the terminal in order to initialize a repo

Reply:
Right, but that's a one time deal, after which the seed is stored (and encrypted if you want) in the repo. The seed isn't needed to start the daemon/node, here you're just using it to choose the repo path.

Reply:
Yea makes sense. This also applies to migrate. I'll update it now.

Reply:
sweet

Reply:
done.